Resources/docs/index.rst

```diff
@@ -0,0 +1,378 @@
+Connector installation
+======================
+
+Add ConnectionsBundle and OXIDConnectorBundle to ``AppKernel.php``
+
+.. code-block:: php
+
+    new ONGR\ConnectionsBundle\ONGRConnectionsBundle(),
+    new ONGR\OXIDConnectorBundle\ONGROXIDConnectorBundle(),
+
+..
+
+Then in your bundle you will need to extend ``Documents`` and ``Entities`` provided in OXIDConnectorBundle
```

Comment:
Documents also need extending :) they are not declared as abstract but will not be added to ES just like entity mapped super classes.